src/main/java/seedu/address/commons/events/model/DeskBoardChangedEvent.java

```diff
@@ -0,0 +1,19 @@
+package seedu.address.commons.events.model;
+
+import seedu.address.commons.events.BaseEvent;
+import seedu.address.model.ReadOnlyDeskBoard;
+
+/** Indicates the DeskBoard in the model has changed*/
```

Comment:
Please add:
//@@author YuanQQLer 
above /** Indicates the DeskBoard in the model has changed*/